lib/assembly/cache.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Assembly.Cache do
+  @moduledoc """
+  Wraps get and puts to cache.
+  Implements the Cachex reactive and proactive cache warming.
+  """
+
+  use Cachex.Warmer
+
+  alias Assembly.Events
+
+  @six_hours 60 * 6
+
+  @impl true
+  def execute(_state) do
+    Events.request_quantity_update()
+    :ignore
+  end
+
+  def fallback(component_id) do
+    Events.request_quantity_update([component_id])
+    :ignore
+  end
+
+  def get(component_id),
+    do: Cachex.get(__MODULE__, component_id)
+
+  @impl true
+  def interval,
```

Comment:
Little nitpick, but I would put this right under `execute/1`

Reply:
We do that elsewhere? I usually group by impl, visibility, then alphabetically. I'll change my style

Reply:
In any code I've written I've done it this way. We discussed it way back in the early days, no biggie.